db/sql/00_msar.sql

```diff
@@ -948,6 +948,54 @@ FROM (
 $$ LANGUAGE SQL STABLE;
 
 
+CREATE OR REPLACE FUNCTION msar.get_role(rolename text) RETURNS jsonb AS $$/*
```

Comment:
We should make it super explicit that this function only returns information about roles which are _direct_ members of the given role. This is in contrast to `pg_has_role(oid, oid, privilege)`. To be honest, I'd rather use something that calls that system function, and just get the direct and indirect members all at once.

Reply:
After discussing with @pavish , we don't need indirect members where this function is used. With that said, this means the term `"members"` as used here is different from the standard usage in PostgreSQL, so we should make sure that's noted in the documentation.

Reply:
Specified in the description in `role_info_table()`